<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	font: 12pt &quot;맑은 고딕&quot;;
}
QLabel {
	font: 9pt &quot;맑은 고딕&quot;;
	font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>사원 정보 조회</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lbl_info">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;사원 정보&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>88</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;DB에서 불러오기&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmb_db_list">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_load_db">
              <property name="text">
               <string>불러오기</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tbl_data">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_load_excel">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>엑셀 불러오기</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_modify">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>데이터 수정</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_save">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>파일로 저장</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_save_db">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>DB에 저장</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="grb_stat">
            <property name="minimumSize">
             <size>
              <width>860</width>
              <height>290</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>사원 데이터 분석</string>
            </property>
            <widget class="QTabWidget" name="tbl_dept_stat_2">
             <property name="geometry">
              <rect>
               <x>264</x>
               <y>33</y>
               <width>590</width>
               <height>250</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>590</width>
               <height>250</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
              </font>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>부서별 통계 현황</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QTableWidget" name="tbl_dept_stat">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>장기 근속자 명단</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTableWidget" name="tbl_long_term">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>14</x>
               <y>35</y>
               <width>241</width>
               <height>131</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>전체 직원 수</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="ln_total_emp">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>장기 근속자 수</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="ln_total_longterm_emp">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>전체 평균 연령</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="ln_emp_age_avg">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:none</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>최고령 직원</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="ln_oldest_emp">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_10">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>통계 기준 일자</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="ln_ref_date">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>맑은 고딕</family>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>삭제된 사원 정보 조회</string>
       </attribute>
       <widget class="QLabel" name="lbl_info_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>151</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;삭제된 사원 정보&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tbl_deleted_data">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>1156</width>
          <height>446</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
